src/components/management/terminals/terminal.tsx

```diff
@@ -25,9 +31,16 @@ const DEFAULT_MAP_CENTER = latLng(61.1621924, 28.65865865);
 function TerminalSiteComponent({ formType, site, onSave }: Props) {
   const { t } = useTranslation();
   const navigate = useNavigate();
+  const [tabValue, setTabValue] = React.useState("1");
+  const [formState] = React.useState(formType);
+  const handleTabChange = (_event: React.SyntheticEvent, newTabValue: string) => {
```

Comment:
Direct imports instead of namespace ones. Also, why is formState initialized with useState instead of useMemo, if it is only set initially and never after?
```suggestion
  const [tabValue, setTabValue] = useState("1");
  const [formState] = useState(formType);
  const handleTabChange = (_event: SyntheticEvent, newTabValue: string) => {
```